packages/story-editor/src/app/story/utils/isEmptyStory.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { shallowEqual } from '@web-stories-wp/react';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_PAGE_BACKGROUND_COLOR } from '../../../elements/utils/createPage';
+
+const isEmptyStory = (pages) => {
```

Comment:
I know it's still WIP and you have Tests as a todo but since it's in the Review column, mentioning anyway:
When you add tests, would be good to have some unit tests for this util, too.

Reply:
To be honest, something is wrong with ZH, because I just connected the issue and ZH moved it 🤔 

![image](https://user-images.githubusercontent.com/21036224/150535652-7111cb4b-1a6f-4b87-a6c7-7df29c496527.png)

Reply:
Hm, okay. I think you need to explicitly set it to "In Progress" even when creating a Draft PR -- it moves it to Ready for Review even if creating a Draft (asked @bmattb about it once, he reached out to ZH but didn't get a solution).